discord/handler.py

```diff
@@ -3,14 +3,15 @@
 
 from typing import Optional, List
 from .embeds import Embed
+from .components import View
 
 
 class Handler:
     def __init__(self):
         self.base_url: str = 'https://discord.com/api/v9/'
         self.user_agent: str = "Disthon test library V0.0.1b"
 
-    async def request(self, method: str, dest: str, *, headers: typing.Optional[dict] = None,
+    async def request(self, method: str, dest: str, *, headers: Optional[dict] = None,
```

Comment:
Either remove all typing, or keep the typing here

Reply:
I prefer if he did keep all the typing